src/components/Field/SelectField/SelectField.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, ChangeEvent } from 'react';
+import MenuItem from '@mui/material/MenuItem';
+
+import TextField, { TextFieldProps } from '../TextField';
+
+type SelectFieldOptions = {
+  value: string;
+  label: string;
+};
+
+export type SelectFieldProps = {
+  options: SelectFieldOptions[];
+} & TextFieldProps;
+
+const SelectField = ({
+  options,
+  ...otherProps
+}: SelectFieldProps): JSX.Element => {
+  const [selectedValue, setSelectedValue] = useState('');
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setSelectedValue(event.target.value);
+  };
+
+  return (
+    <TextField
+      variant='outlined'
+      select
+      id='select'
```

Comment:
nit: double quotes for props (ugh, really need to set up prettier here properly...)